Solutions/CohesitySecurity/Tests/az.test.py

```diff
@@ -1,16 +1,65 @@
 #!/usr/bin/env python3
 
+import subprocess
 import json
+import time
 from az import *
+from helios import *
 import os
 
 os.chdir(os.path.dirname(os.path.abspath(__file__)))
 f = open('../cohesity.json',)
 data = json.load(f)
 resource_group = data['resource_group']
 workspace_name = data['workspace_name']
-playbook_name = data['playbook_name']
 apiKey = data['apiKey']
```

Comment:
Could we please validate that none of these values are empty so that we fail fast and the user knows that their environment is not configured?

Reply:
we don't need to validate/test against test codes, this is over test.
the user just need to fill the cohesity.json file, which is also used during cli deployment.           
it is very rarely that the user would leave some attributes empty.

Reply:
This kind of defensive programming is used all across Cohesity. Could you tell me if not having 'apiKey' in the json will exit here immediately? Or will it proceed further with empty value? If it proceeds without any empty value, then I would insist on assertion. If it fails immediately, then I'll leave it up to you whether to add assert on empty or not.

Reply:
could you see my comments above?